js/src/sdk/models/actions.ts

```diff
@@ -180,15 +184,22 @@ export class Actions {
      */
     async list(data: GetListActionsData = {}): Promise<ActionsListResponseDTO> {
         try {
+
+            let apps = data.apps;
+            if(data?.filterByAvailableApps){
```

Comment:
We should throw an error here if the user ends up supplying both `apps` and `filterByAvailableApps` as params so they doesn't get confused why the `apps` parameters was ignored as per the logic below.